src/helper/main.js

```diff
@@ -79,6 +79,74 @@ import { logAndDie, helpMessage, databases } from './helperUtil';
             console.log('Finished!');
             break;
         }
+        case 'firebase': {
+            try {
+                console.log('Reading info...');
+                const info = JSON.parse(await fs.readFile('./data/firebase.json'));
+                const ownerOrKeyRegex = /^(?:\w+-\w+-\w+)|(?:owner)$/;
```

Comment:
maybe don't use + but {4} instead, as steamkeys are formatted more specificly

Reply:
Well yeah but the only other values are "owner" and words without dashes so we should be ok